XLS-0033d-CFTs/README.md

```diff
@@ -802,8 +840,32 @@ A JSON object representing a dictionary of accounts to CFToken objects. Includes
 
 Used to continue querying where we left off when paginating. Omitted if there are no more entries after this result.
 
-
+ ### 1.7 Free CFTs
+When a holder creates a `CFTokenPage`, if the holder owns at most 2 items in the ledger including the new page, the account's owner reserve is treated as zero instead of the normal amount. This is following the status quo of how free trustlines work today.
```

Comment:
It's a product requirement, we would want to incentive holders to use CFT, also less learning curve, as they don't need to care about reserve requirements. Giving them free CFTokenPage would mean that they can get 32 CFTs free of reserves

Reply:
I don't love this "requirement" for a few reasons:

1. It shields users from reserve requirements, which is likely to cause confusion -- once a user exceeds whatever arbitrary "free" limit we give them -- but only later after a user has been using CFTs for awhile. I would prefer that users encounter reserve fees right away, because this is something that users probably need to be aware of anyway.
2. Reserve fees exist for a reason, which is to pay for space on-ledger. There's a decent chance that a large group of people hold fewer than 32 CFTs, which means there's a decent chance that a large group of people make _many_ CFTs and none of them pay reserve fees. Put another way, I worry that this "requirement" essentially exempts CFTs from reserve fees, which seems counter-productive (assuming reserve fees are there for a reasonable purpose, which I believe to be true).
3. Last but not least, this likely introduces unnecessary code/implementation complexity around ledger fee computation.

That said, I'm going to resolve this thread to unblock this PR. We can (potentially) debate this in another PR.